PayPalPayoutsSdk/PayPalPayoutsSdk.csproj

```diff
@@ -2,7 +2,7 @@
 
   <PropertyGroup>
     <TargetFramework>netstandard2.0</TargetFramework>
-    <PackageVersion>1.0.3</PackageVersion>
+    <PackageVersion>1.0.1</PackageVersion>
```

Comment:
this is not a patch fix, can this be treated as a minor version

Reply:
changed